spec/services/importers/doc/alias_spec.rb

```diff
@@ -1,9 +1,22 @@
 require 'rails_helper'
+require 'services/importers/doc/shared_specs'
 
 RSpec.describe Importers::Doc::Alias do
-  describe '#perform' do
-    it 'imports the data' do
-      skip
+  it_behaves_like('doc_importer') do
+    let(:class_to_import) { ::Doc::Alias }
+    let(:sample_file) do
+      ' 0000123482BELTON                        RICHARD                       LEWIS                         JR'
+    end
```

Comment:
What would you think about moving this to either a Fixture or a Factory? It is mainly for aesthetics. But I could also see a scenario where a single character is deleted and it is hard to find the bug.

Reply:
yeah good point..
I typically use factories but not sure how I feel about taking the factory and converting it in to the proper format (seems a bit clunky but might not be bad). When I use factories I'll reference the created value in my expected assertion. 
With fixtures you just hardcode the values even though they're coming from a separate file?

Reply:
Resolving for now will try to come back to when I have more time!